pkg/provisioning/cbnt/tools.go

```diff
@@ -289,10 +289,29 @@ func FindAdditionalIBBs(imagepath string) ([]bootpolicy.IBBSegment, error) {
 	}
 
 	img, err := cbfs.NewImage(image)
+	// If this returns an error, we assume it's an UEFI image
 	if err != nil {
-		return nil, err
+		// To be sure the image file is closed before reading from it again
+		image.Close()
+		img, err := ioutil.ReadFile(imagepath)
+		if err != nil {
+			return ibbs, err
+		}
+		fitentries, err := fit.GetEntries(img)
+		if err != nil {
+			return ibbs, err
+		}
+		for _, entry := range fitentries {
+			if entry.GetType() == fit.EntryTypeBIOSStartupModuleEntry {
+				ibb := bootpolicy.NewIBBSegment()
+				ibb.Base = uint32(entry.GetHeaders().Address.Pointer())
+				ibb.Size = entry.GetHeaders().Size.Uint32() << 4
+				ibbs = append(ibbs, *ibb)
+			}
+		}
+		return ibbs, nil
 	}
-
+	// From here we consider it is a coreboot image
```

Comment:
better check for FMAP in BIOS region. fiano should support that ccheck

Reply:
cbfs.NewImage(...) checks for FMAP named COREBOOT. Why is that not sufficient?

Reply:
That's sufficient and even desirable for buildtesting in coreboot where no IFD is present.